event_sale_session/models/sale_order_line.py

```diff
@@ -25,16 +26,16 @@ def _onchange_event_id(self):
             self.event_session_id = event.session_ids[0]
         elif not event.use_sessions or event != self.event_session_id.event_id:
             self.event_session_id = False
-        return super()._onchange_event_id()
+        return super()._compute_event_ticket_id()
```

Comment:
⚠️ : This doesn't seem right.

This method is named `_onchange_event_id` but it's calling super `_compute_event_ticket_id`

Reply:
After checking, I think we'd want to align with what Odoo's made for `event_ticket_id`, [making it a computed field (store=True, readonly=False, precompute=True)](https://github.com/odoo/odoo/blob/68c6416c03c31a04b4458f8a64886215d14675ba/addons/event_sale/models/sale_order.py#L69-L72).

So, in our case, `event_session_id` should be a computed method with its own `_compute_event_session_id`

Reply:
Thanks for all the comments 😃.
I've checked it and made `event_session_id` a compute field, similar to `event_ticket_id`. Have I done it right?

Reply:
If yo make the computed field you don't need to overwrite the _compute_event_ticket_id

Reply:
Got it, can you check it please?